attending/index.py

```diff
@@ -0,0 +1,19 @@
+import csv
+from pathlib import Path
+import os
+
+
+def load_index(file: Path):
```

Comment:
so I know you like Paths, but once you eliminate the assertion check below, there's no reason to limit `file` to a Path, strings are fine here, too. See [the PEP 519 specification](https://www.python.org/dev/peps/pep-0519/) which suggests the path annotation of `path: Union[PathLike, str, bytes]` for the first parameter of os.fspath, for example.

Also, calling a variable `file` is <strike>poor practice for shadowing ....</strike> oops, I forgot that Python 3 got rid of the `file` builtin... it still looks weird to me... but that probably more of a "who moved my cheese" resistance that I need to work through.

```suggestion
def load_urls(file):
```